data/remnant/remnant 2 cognizance.txt

```diff
@@ -1383,7 +1372,7 @@ mission "Remnant: Cognizance 32"
 			branch ssilvidaemeraldsword
 				has "Sheragi Archaeology: Epilogue: done"
 			`	A hologram of Ssil Vida appears in the air over the table, then fades away to leave a large highlighted object with a network of lines reaching around the station. "As Gavriil has told you, the station's primary power source appears to be a massive fusion reactor. As best we can tell, it is quite a simple design, albeit one built for totally automated functioning and bigger than the largest reactors ever built by humanity. Unfortunately, it has consumed its entire fuel supply and is currently using up the last of the fuel it is capable of extracting by refining spent fuel. Our current estimates is that it is producing less than ten percent of its nominal power. This is sufficiently low that it has already ceased powering the majority of the station's systems." He gestures to Chilia.`
-			`	"Based on our analysis, the reactor appears to use a very specific isotope. Whether it is truly needed or others would suffice is unknown, but for now we are inclined to provide it with an exact match. While we will be able to produce it eventually, it will take time that we do not have." The diagram of Ssil Vida vanishes and is replaced by a map of the galaxy, which focuses in on <system>, near the juncture of the Rim and the South. "Our records indicate that the most easily accessible deposits are on <planet>." Chilia turns to you. "As you are from ancestral space and are most familiar with the people there, we would like you to travel there and bring back 58 cubic meters of processed fuel for the reactor. Can you do this?"`
+			`	"Based on our analysis, the reactor appears to use a very specific isotope. Whether it is truly needed or others would suffice is unknown, but for now we are inclined to provide it with an exact match. While we will be able to produce it eventually, it will take time that we do not have." The diagram of Ssil Vida vanishes and is replaced by a map of the galaxy, which focuses in on <system>, near the juncture of the Rim and the South. "Our records indicate that the most easily accessible deposits are on <planet>." Chilia turns to you. "As you are from ancestral space and are most familiar with the people there, we would like you to travel there and bring back 58 tons of processed fuel for the reactor. Can you do this?"`
```

Comment:
```suggestion
			`	"Based on our analysis, the reactor appears to use a very specific isotope. Whether it is truly needed or others would suffice is unknown, but for now we are inclined to provide it with an exact match. While we will be able to produce it eventually, it will take time that we do not have." The diagram of Ssil Vida vanishes and is replaced by a map of the galaxy, which focuses in on <system>, near the juncture of the Rim and the South. "Our records indicate that the most easily accessible deposits are on <planet>." Chilia turns to you. "As you are from ancestral space and are most familiar with the people there, we would like you to travel there and bring back 58 cubic meters of processed fuel for the reactor. Can you do this?"`
```

While the rest of humanity may use tons, the Remnant do not. Given that the word "ton" does, in fact, refer to 1000kg (or 1 cubic meter of water), the reference here to cubic meters does not conflict with the use of ton elesewhere. (Yes, I am well aware that ton *can* refer to about a dozen other potential quantities, but humanity is standardizing on metric, so I strongly suspect that our "ton" is specifically the 1000kg one).

Reply:
> Given that the word "ton" does, in fact, refer to 1000kg (or 1 cubic meter of water), the reference here to cubic meters does not conflict with the use of ton elesewhere.

The player should not be expected to guess at what the meaning of these units is. If you want to use something other than `tons`, it should just be `units`, since that is a generic term.

Reply:
There is also the fact that this is within dialog. NPCs are allowed to say things that may confuse the player and/or not conform to the rest of the game.